test/e2e/replica_set_readiness_probe/replica_set_readiness_probe_test.go

```diff
@@ -16,6 +16,10 @@ func TestMain(m *testing.M) {
 	f.MainEntry(m)
 }
 
+func init() {
```

Comment:
let's try to avoid init functions when possible

Reply:
Will do!